GPL/Events/Process/Probe.bpf.c

```diff
@@ -226,6 +257,195 @@ int tracepoint_syscalls_sys_exit_setsid(struct trace_event_raw_sys_exit *args)
     return 0;
 }
 
+SEC("tp_btf/module_load")
+int BPF_PROG(module_load, struct module *mod)
+{
+    if (ebpf_events_is_trusted_pid())
+        goto out;
+
+    struct ebpf_process_load_module_event *event = get_event_buffer();
+    if (!event)
+        goto out;
+
+    event->hdr.type = EBPF_EVENT_PROCESS_LOAD_MODULE;
+    event->hdr.ts   = bpf_ktime_get_ns();
+
+    struct task_struct *task = (struct task_struct *)bpf_get_current_task();
+
+    if (is_kernel_thread(task))
+        goto out;
```

Comment:
```suggestion
    struct task_struct *task = (struct task_struct *)bpf_get_current_task();

    if (is_kernel_thread(task))
        goto out;

    struct ebpf_process_load_module_event *event = get_event_buffer();
    if (!event)
        goto out;

    event->hdr.type = EBPF_EVENT_PROCESS_LOAD_MODULE;
    event->hdr.ts   = bpf_ktime_get_ns();

```

Rest of the source checks task before getting event, I think we should keep the same pattern, pattern are goods and make for a healthy breakfast.

Reply:
done